cypress-io/cypress
# chore: updating log to use createdAtTimestamp and updatedAtTimestamp

### Additional details

* Renamed the `timestamp` field to `updatedAtTimestamp`
* Added `createdAtTimestamp` field

### Steps to test

Run a protocol test and verify the `updatedAtTimestamp` and `createdAtTimestamp` fields are in the database.

### How has the user experience changed?


### PR Tasks


- [n/a] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
